standalone/test/include/psen_scan_v2_standalone/data_conversion_layer/io_pin_data_helper.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2021 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#ifndef PSEN_SCAN_V2_STANDALONE_TEST_IO_PIN_DATA_HELPER_H
+#define PSEN_SCAN_V2_STAND
+
+#include <string>
+#include <vector>
+#include <algorithm>
+
+#include "psen_scan_v2_standalone/data_conversion_layer/io_pin_data.h"
+#include "psen_scan_v2_standalone/io_state.h"
+
+namespace psen_scan_v2_standalone_test
+{
+using namespace psen_scan_v2_standalone;
+namespace dmi = data_conversion_layer::monitoring_frame::io;
```

Comment:
@agutenkunst Do you consider namespace alias bad style in general (I guess we agree that an abbreviation should be avoided)? Recently, I tried to avoid `using namespace` and I think a namespace alias is often a good alternative. If it would be only the three types you wrote I would support your suggestion. But there are more constants and functions used from the `io` namespace.

Reply:
so essentially the division into a production and test namespace is the root cause for trouble here...